services/puppetforge/puppetforge-module-quality-score.tester.js

```diff
@@ -0,0 +1,27 @@
+import { isPercentage } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('module quality-score').get('/camptocamp/openssl.json').expectBadge({
+  label: 'score',
+  message: isPercentage,
+})
+
+t.create('module quality score (no ratings)')
+  .get('/camptocamp/openssl.json')
+  .intercept(nock =>
+    nock('https://forgeapi.puppetlabs.com/private/validations')
+      .get('/camptocamp-openssl')
+      .reply(200, {}),
```

Comment:
Surely the invalid response would be {} not []?
Do you have an example of the kind of response we're trying to guard against.

Reply:
Anything that does not validate the schema is good, so an empty array is right I think.